CONTRIBUTING.md

```diff
@@ -70,10 +70,12 @@ guidelines:
 
 ### Commits in your pull-requests should
 
-- Have a useful description prefixed with the package name (E.g.: "foopkg: Add
-  libzot dependency")
+- Have a useful commit subject prefixed with the package name (E.g.: "foopkg:
+  Add libzot dependency").
 - Include Signed-off-by tag in the commit comments.  See: [Sign your
```

Comment:
Please also add that the name must be real and GitHub private email aren't accepted.

Reply:
Done.

Reply:
https://github.com/openwrt/packages/pull/23084

Reply:
@systemcrash, I copied the "real name or known identity" verbatim from https://openwrt.org/submitting-patches#submission_guidelines, and I think this wording is consistent with that discussion on the PR you linked. Please let me know if you're looking for something else.

Reply:
That's fine. That is all that's needed.